regression/test_stress_failovering.py

```diff
@@ -50,6 +50,11 @@ def test_limited_ka(self):
         """
         self.run_test(100)
 
+class RatioFailoveringTempesta502(RatioFailovering):
```

Comment:
Too hurry change. `RatioFailovering` had two tests: `test_limited_ka` and `test_unlimited_ka`. Class `HashFailovering` is derived from `RatioFailovering` , and now  `test_unlimited_ka` is missed in `HashFailovering` .

Reply:
Fixed